node-api/backend/genesis_test.go

```diff
@@ -59,31 +109,17 @@ func setupStateWithGenesisValues(
 	sdkCtx.MultiStore().(storetypes.CacheMultiStore).Write()
 }
 
-func TestGetGenesisData(t *testing.T) {
```

Comment:
diff tripping here. I moved the whole piece of code needed to generate and store CometBFT config into a tmp folder to its own method